Trying again: Fixing peak area, rubberband, and M3 and CRISM io bugs

ryanbanderson opened the pull request:
Closed #222 and opened this to see if it gets travis to work properly.

ryanbanderson commented:
Well, at least this triggered a travis build, but once again it passed and isn't showing up here...
https://travis-ci.org/github/USGS-Astrogeology/PyHAT/builds/671775051

jlaura commented:
@tthatcher95 Can you checkout Travis and see why it might not be returning?

jlaura commented:
@ryanbanderson Looks like you need to rebase @tthatcher95's changes into this PR. You've got 32 files changed, but I think your PR should be 4 files or so?

ryanbanderson commented:
@jlaura ok, I rebased again but that didn't help. I updated USGS dev from USGS master so the CI changes @tthatcher95 has been making no longer show as changes here. There are still 25 changes because I deleted some outdated LIBS example data and uploaded a new file for testing, caught and fixed a few typos, and also deleted some stuff that I thought was removed a while ago - not sure if the first rebase somehow brought it back or if it was never actually removed.

Still not clear why travis isn't working and I'm not sure what to do about it.

ryanbanderson commented:
Closing this and opening a new one (again) to see if travis will work.
